<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>fl_0</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Geraffel Village :: B33rn4ry Counter Registration Desk</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelLogo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>681</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>600</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>exit.png</normaloff>exit.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="newUserButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>600</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>STRG+N</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>newUser.png</normaloff>newUser.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>600</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>F5</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>refresh.png</normaloff>refresh.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="delUserButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>600</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>rm $user</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>deleteUser.png</normaloff>deleteUser.png</iconset>
    </property>
   </widget>
   <widget class="QTableWidget" name="userTable">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>691</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashLine</enum>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>150</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QLabel" name="labelMessage">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>480</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="serviceButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>600</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Service</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>942</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNewUser"/>
    <addaction name="separator"/>
    <addaction name="actionRefreshUserTable"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="toolTip">
    <string>Status bar</string>
   </property>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset>
     <normaloff>exit.png</normaloff>exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="iconText">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNewUser">
   <property name="icon">
    <iconset>
     <normaloff>../B33rn4ry Counter/newUser.png</normaloff>../B33rn4ry Counter/newUser.png</iconset>
   </property>
   <property name="text">
    <string>New User</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionRefreshUserTable">
   <property name="icon">
    <iconset>
     <normaloff>../B33rn4ry Counter/refresh.png</normaloff>../B33rn4ry Counter/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh User Table</string>
   </property>
   <property name="toolTip">
    <string>Refresh User Table</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
